pkg/handlers/requests.go

```diff
@@ -64,5 +64,7 @@ type ServiceHost struct {
 	Host  string   `json:"host"`
 }
 type ScanRequest struct {
-	HostIds []string `json:"host_ids"`
+	HostId     int     `json:"host_id"`
```

Comment:
struct field `HostId` should be `HostID` [(ST1003)](https://staticcheck.dev/docs/checks/#ST1003)

Please don't forget we have an integrated audit static and vulnerability checker in the app, which you can run using `make audit` to avoid these kinds of errors.

```suggestion
	HostID     int     `json:"host_id"`
```

Reply:
I forgot to use that audit,  I will do it before sending to you